.github/ISSUE_TEMPLATE.md

```diff
@@ -14,18 +14,16 @@
 #### Browser Information
 <!-- Describe your workspace in which you are having issues-->
 
-* Browser Name, Version: 
-* Operating System: 
-* Mobile, Desktop, or Tablet: 
+* Browser Name, Version:
+* Operating System:
+* Mobile, Desktop, or Tablet:
 
 #### Your Code
 <!-- If relevant, paste all of your challenge code in here -->
 ```js
- 
- 
- 
+
+
+
```

Comment:
@Greenheart There's 3 new lines between the backticks, no? Not sure why there should be a white space left there? Is there a reason why code shouldn't start at the first character of the new line?